packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/label.overrides

```diff
@@ -53,7 +53,7 @@
 }
 
 /* STATE | With Tag | Disabled */
-.ui.button.disabled .empty.circular.label {
+.ui.empty.circular.label.disabled {
```

Comment:
instead of targeting `ui.empty.circular.label.disabled` you should target `ui.tag.disabled`